pwiz_tools/Skyline/TestFunctional/ExportIsolationListTest.cs

```diff
@@ -467,6 +473,60 @@ private string FieldSeparate(params object[] valuesIn)
             }
             return sb.ToString();
         }
+

```

Comment:
Thanks for adding a test. Let's make sure we get the Delta RT field right, and I see another opportunity to use your constant instead of 0.1 as a magic value below.